src/main/java/net/mcreator/blockly/datapack/BlocklyToJSONTrigger.java

```diff
@@ -48,6 +49,7 @@
 public class BlocklyToJSONTrigger extends BlocklyToCode {
 
 	private static final Logger LOG = LogManager.getLogger("Blockly2JSONTrigger");
+	private static final BlocklyEditorType editorType = BlocklyEditorType.JSON_TRIGGER;

```

Comment:
`editorType` is argument of BlocklyToCode so redifining it here as static shadows that variable. Use property of object instead

Reply:
`editorType` is an argument of BlocklyToJava only. It was never implemented inside BlocklyToCode as it meant adding support for JSON triggers (and I wanted to avoid this at the beginning). However, now that they use it, I generalize it argument to BlocklyToCode.